src/TurbulenceClosures/turbulence_closure_implementations/ri_based_vertical_diffusivity.jl

```diff
@@ -106,8 +106,8 @@ with_tracers(tracers, closure::FlavorOfRBVD) = closure
 
 # Note: computing diffusivities at cell centers for now.
 function DiffusivityFields(grid, tracer_names, bcs, closure::FlavorOfRBVD)
-    κ = Field{Center, Center, Face}(grid)
-    ν = Field{Center, Center, Face}(grid)
+    κ = Field((Center, Center, Face), grid)
+    ν = Field((Center, Center, Face), grid)
```

Comment:
Why?

Reply:
it is always the same motivation. We can revert everything if we agree on it, but it would be nice to have a once consistent way to define a field.